utf8.h

```diff
@@ -378,10 +378,12 @@ are in the character. */
 
 /* This defines the bits that are to be in the continuation bytes of a
  * multi-byte UTF-8 encoded character that mark it is a continuation byte.
- * This turns out to be 0x80 in UTF-8, 0xA0 in UTF-EBCDIC.  (khw doesn't know
- * the underlying reason that B0 works here, except it just happens to work.
- * One could solve for two linear equations and come up with it.) */
-#define UTF_CONTINUATION_MARK       (UTF_IS_CONTINUATION_MASK & 0xB0)
+ * A MASK clears the bits you don't want, and a MARK sets the ones you do want.
+ * For this MARK, all the bits that aren't reserved for the byte's payload
+ * (~UTF_CONTINUATION_MASK) are set to 1 except the next to highest one (BF).
+ * This yields 1000 0000 (0x80) for ASCII (a 6 bit payload), and 1010 0000
+ * (0xA0) for UTF-EBCDIC (a 5 bit payload.). */
```

Comment:
I don't think the new explanation adds much.

From an implementers PoV UTF-8 uses the bit patterns it does because those are the patterns selected by the UC.

Similarly for UTF-EBCDIC.

The constant change really just swaps one magic number (0xB0) for another magic number (0xBF).